python/tests/website/test_web_equipment.py

```diff
@@ -0,0 +1,731 @@
+import pytest
+import time
+import os
+import requests
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from web_test_utils import login_to_equipment, login_with_password
+
+"""
+The test_web_equipment.py file contains comprehensive tests for PiFinder's equipment management web interface.
+
+Test Overview
+
+The test suite validates PiFinder's equipment management functionality through automated browser testing using Selenium
+WebDriver. All tests authenticate with the default password "solveit" and interact with the equipment configuration
+interface at localhost:8080/equipment.
+
+Core Interface Tests
+
+Equipment Navigation: Tests verify that navigation to the equipment page works correctly from the home page using
+the "Equipment" link in both desktop and mobile navigation menus.
+
+Table Structure: Tests validate that both the Instruments and Eyepieces tables are present with correct column
+headers and proper table structure for displaying equipment data.
+
+Equipment Management Tests
+
+Instrument Management: Tests cover adding new instruments with test data, verifying they appear in the table with
+correct values, and then removing the test entries to maintain clean test state. Tests validate form submission,
+data persistence, and proper table updates.
+
+Eyepiece Management: Similar comprehensive testing for eyepiece management including add/edit/delete operations,
+form validation, and table updates.
+
+Active Equipment Selection
+
+Active Instrument Selection: Tests verify that users can select active instruments using the radio button interface,
+ensuring the selection is properly reflected in the UI and persists correctly.
+
+Active Eyepiece Selection: Similar testing for eyepiece selection functionality, validating the radio button
+interface and selection persistence.
+
+Technical Implementation
+
+Authentication: Uses the same login flow as other web interface tests with the default "solveit" password.
+Form Validation: Tests check for proper form structure, input field validation, and error handling.
+Data Persistence: Verifies that equipment data persists correctly across page refreshes and navigation.
+Clean Test State: Tests clean up after themselves by removing test entries to avoid interference between runs.
+
+Infrastructure: Uses the same Selenium Grid setup as other web tests with automatic skipping when unavailable.
+(Summary created by Claude Code)
+"""
+
+
+@pytest.fixture(scope="session")
+def shared_driver():
```

Comment:
Can this also be moved into the web_test_utils.py to be used across all the test modules?